Tasks/DockerV2/dockerpush.ts

```diff
@@ -79,3 +99,58 @@ export function run(connection: ContainerConnection, outputUpdate: (data: string
 
     return promise;
 }
+
+async function publishToImageMetadataStore(connection: ContainerConnection, imageName: string, tags: string[], digest: string, dockerFilePath: string): Promise<any> {
+    const imageUri: string = getResourceName(imageName, digest);
+    const dockerFileContent: string = fs.readFileSync(dockerFilePath, 'utf-8').toString();
```

Comment:
We should read the docker file in getBaseImageName method itself. Maybe have another overload of it.

Reply:
Added another function getBaseImageNameFromDockerFileContent in containerImageUtils which reads the docker file and calls getBaseImageName